.travis.yml

```diff
@@ -2,10 +2,10 @@ sudo: required
 
 matrix:
   include:
-     - env: OC_CLUSTER_UP=true OC_VERSION="v3.6.0" COMMIT_ID="c4dd4cf" CATEGORY="org.arquillian.cube.kubernetes.impl.requirement.RequiresKubernetes,org.arquillian.cube.openshift.impl.requirement.RequiresOpenshift"
      - env: OC_CLUSTER_UP=true OC_VERSION="v3.6.1" COMMIT_ID="008f2d5" CATEGORY="org.arquillian.cube.kubernetes.impl.requirement.RequiresKubernetes,org.arquillian.cube.openshift.impl.requirement.RequiresOpenshift"
-     - env: OC_CLUSTER_UP=true OC_VERSION="v3.7.0" COMMIT_ID="7ed6862" CATEGORY="org.arquillian.cube.kubernetes.impl.requirement.RequiresKubernetes,org.arquillian.cube.openshift.impl.requirement.RequiresOpenshift"
      - env: OC_CLUSTER_UP=true OC_VERSION="v3.7.1" COMMIT_ID="ab0f056" CATEGORY="org.arquillian.cube.kubernetes.impl.requirement.RequiresKubernetes,org.arquillian.cube.openshift.impl.requirement.RequiresOpenshift"
+     - env: OC_CLUSTER_UP=true OC_VERSION="v3.7.2" COMMIT_ID="282e43f" CATEGORY="org.arquillian.cube.kubernetes.impl.requirement.RequiresKubernetes,org.arquillian.cube.openshift.impl.requirement.RequiresOpenshift"
+     - env: OC_CLUSTER_UP=true OC_VERSION="v3.9.0-alpha.3" COMMIT_ID="78ddc10" CATEGORY="org.arquillian.cube.kubernetes.impl.requirement.RequiresKubernetes,org.arquillian.cube.openshift.impl.requirement.RequiresOpenshift"
```

Comment:
How about `v3.9.0-alpha.4`?

Reply:
We don't have openshift-origin-client-tools-v3.7.2-282e43f-linux-64bit.tar.gz available for it
https://github.com/openshift/origin/releases/tag/v3.9.0-alpha.4